lib/apple/index.js

```diff
@@ -32,14 +46,27 @@ function parseResult(result) {
 		throw error;
 	}
 
-	var receipt = result.receipt;
+	var productId = getReceiptFieldValue(result.receipt, 'product_id');
+	var transactionId = getReceiptFieldValue(result.receipt, 'transaction_id');
+
+	/* jshint camelcase:false */
+    if (result.hasOwnProperty('latest_receipt_info') && result.latest_receipt_info[0]) {
+        latestReceiptInfo = result.latest_receipt_info.sort(function (a, b) {
+            a = parseInt(a.original_purchase_date_ms);
+            b = parseInt(b.original_purchase_date_ms);
```

Comment:
No worries, your helping me to develop good practices --appreciate it! Plus it doesn't hurt to follow Mozilla via "Always specify this parameter to eliminate reader confusion and to guarantee predictable behavior."